README.rst

```diff
@@ -98,10 +148,50 @@ For tracing, use the @signalfx_lambda.is_traced decorator
     def handler(event, context):
         # your code
 
-The decorators can be used individually or together.
+
+Step 5: Send custom metrics from a Lambda function
+-------------------------------------------------------
+
+1. To send custom metrics from a Lambda function, run the following commands in your command line: 
```

Comment:
Please change "run the following commands in your command line:" to "include the following code in your function:" or equivalent. The same remark for Step 6.1. Except for that, I am OK with merging this changes.

Reply:
This has been updated.